tests/test_memory.py

```diff
@@ -0,0 +1,56 @@
+from happy_bank_core.data.memory import AccountNotFoundException, MemoryConnector
+from happy_bank_core.logic.account import Account
+import pytest
+
+
+class TestMemory:
```

Comment:
And module name should be: test_memory_connector.py

```suggestion
class TestMemoryConnector:
```